lucene/analysis/common/src/java/org/apache/lucene/analysis/hunspell/Stemmer.java

```diff
@@ -654,7 +646,7 @@ private boolean checkCondition(
                   true,
                   circumfix,
                   caseVariant));
-        } else if (dictionary.complexPrefixes == false && dictionary.twoStageAffix) {
+        } else if (!dictionary.complexPrefixes && dictionary.twoStageAffix) {
```

Comment:
I don't care much but I know some other committers prefer the verbose version (== false).

Reply:
I prefer shorter versions (especially given the short column limit of spotlessJava), and IntelliJ highlights this by default, but if you insist, I can disable the highlighting and live with it. What's the rule exactly, write `x == false` instead of `!x` everywhere?

Reply:
So do I. But @romseygeek has a different preference. I think you can follow your own gut feeling here - there are no explicit rules that we enforce.

Reply:
Then I'd shorten this :)